library/Zend/Http/Client.php

```diff
@@ -1273,12 +1273,13 @@ protected function _prepareBody()
         }
         // If mbstring overloads substr and strlen functions, we have to
         // override it's internal encoding
-        trigger_error("PHP 7.2 Compatibility Alert:\n\tWARNING: INI directive 'mbstring.func_overload' is deprecated since PHP 7.2".sprintf(" (%s::%s)", __FILE__, __LINE__)."\n\t".implode("\n\t", array_map(function ($item) { return sprintf("%s::%s", $item['file'], $item['line']); }, debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS))), E_USER_WARNING);
-        if (function_exists('mb_internal_encoding') &&
-           ((int) ini_get('mbstring.func_overload')) & 2) {
```

Comment:
this alerts that an ini param is deprecated as of PHP 7.2.
* i _believe_ it's safe to use in PHP 7.2, but that its promised to be removed in a later version.
* luckily, our ansible `php.ini`s do not seem to set this, so we don't seem to rely on the affected code
* if i'm right, it feels like this could be handled with an "if PHP_VERSION <= 7.2 { alert that we won't fix } else { alert like all the rest }".
* i guess the biggest concern is to verify that we don't use the ini parameter anywhere, and that we don't begin to use it (since it's going away).

how we see it used in kibana:
```
PHP 7.2 Compatibility Alert:
	WARNING: INI directive 'mbstring.func_overload' is deprecated since PHP 7.2 (/export/sites/ifbyphone.com/releases/20190509084540/vendor/shardj/zf1-future/library/Zend/Http/Client.php::1277)
	/export/sites/ifbyphone.com/releases/20190509084540/vendor/shardj/zf1-future/library/Zend/Http/Client.php::1061
	/export/sites/ifbyphone.com/releases/20190509084540/controller/process_call_recordings.php::371
	/export/sites/ifbyphone.com/releases/20190509084540/controller/process_call_recordings.php::253
```

Reply:
I believe you are right - that it is deprecated, but not removed in 7.2.  So not a biggie yet.